joi/index.d.ts

```diff
@@ -121,17 +121,17 @@ export interface IPOptions {
 }
 
 export interface ValidationError extends Error {
-    message: string;
+    isJoi: boolean;
     details: ValidationErrorItem[];
-    simple(): string;
-    annotated(): string;
+    annotate(): string;
+    _object: any;
 }
 
 export interface ValidationErrorItem {
     message: string;
     type: string;
     path: string;
-    options?: ValidationOptions;
```

Comment:
looks like this does exist, see https://github.com/hapijs/joi/blob/a2b0b8d43b6b8c0b0b556ceeb1537c6a1d67be4f/lib/errors.js#L55